external/mqtt/pom.xml

```diff
@@ -16,65 +16,71 @@
   ~ limitations under the License.
   -->
 
```

Comment:
This file should not change completely like this. I think there has been some incorrect changes in the indentation. Please revert/fix this. If there are any changes necessary related to test scope (for unit test) only those lines should change.